src/test/resources/features/convertAudio/audio.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Audio Converter
+
+  In order to convert a file
```

Comment:
Please consider changing this statement to something like: `In order to convert an audio file`.